Base/res/html/misc/text-decoration.html

```diff
@@ -4,15 +4,17 @@
 <title>text-decoration test</title>
 <style>
 .overline { text-decoration: wavy blue overline; }
-.underline { text-decoration: double red underline; }
-.strikethrough { text-decoration: dotted green line-through; }
+.underline { text-decoration: red underline double; }
```

Comment:
Isn't the old way also technically correct?

Reply:
Yeah, this is just about testing that it still works in different orders.